### PR TITLE
Add graphql

### PR DESCRIPTION
**Pull request by Bdthomson**
Run `yarn graphql`.

### CODE REVIEW

**Comment by Bdthomson** [comment]
Won't build without something like this - https://github.com/connexta/kanri/blob/master/package.json#L55

File: graphql/graphql.js [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+    .map(attrs => renameKeys(k => queryTemplates[k], attrs))
+    .map(({ modified, created, ...rest }) => {
+      return {
+        ...rest,
+        created: new Date(created).toISOString(),
+        modified: new Date(modified).toISOString(),
+      }
+    })
+  const status = {
+    // count: Int
```
Comment (stevenmalmgren) [inline graphql/graphql.js]: ✏️ Why are these here?
